sdk/tests/conformance2/extensions/ext-color-buffer-float.html

```diff
@@ -427,11 +427,7 @@
         if (enabled) {
             shouldBe("gl.checkFramebufferStatus(gl.FRAMEBUFFER)", "gl.FRAMEBUFFER_COMPLETE");
             gl.copyTexImage2D(gl.TEXTURE_2D, level, destInternalformat, 0, 0, width, height, 0);
-            if (destInternalformat == gl.RGB16F || destInternalformat == gl.RGB32F) {
-                wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, "CopyTexImage2D should fail.");
-            } else {
-                wtu.glErrorShouldBe(gl, gl.NO_ERROR, "CopyTexImage2D should succeed.");
-            }
+            wtu.glErrorShouldBe(gl, gl.NO_ERROR, "CopyTexImage2D should succeed.");
         } else {
```

Comment:
This change is incorrect. RGB{16,32}F formats are not color-renderable. Neither the GL_EXT_color_buffer_float extension nor the WebGL wrapper make them so. This test needs to remain as is.